apps/pi-drone-detector/azure-ml/train.py

```
@@ -0,0 +1,125 @@
+    parser.add_argument('--epochs', type=int, default=100, help='Number of epochs')
+    parser.add_argument('--imgsz', type=int, default=320, help='Image size (square)')
+    parser.add_argument('--batch', type=int, default=16, help='Batch size')
+    parser.add_argument('--output', type=str, default='./outputs', help='Output directory')
+    parser.add_argument('--model', type=str, default='yolov5n.pt', help='Base model')
+    parser.add_argument('--device', type=str, default='0', help='CUDA device')
+    args = parser.parse_args()
+
+    # Import here to allow --help without ultralytics installed
+    from ultralytics import YOLO
+
+    print(f"Starting training with config:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)